python/tests/test_simulation.py

```diff
@@ -336,6 +340,9 @@ def get_ref_field_point(sim):
         if chunk_sim:
             chunk_layout = sim1
 
+        fields_dump_fn = os.path.join(self.temp_dir, 'test_load_dump_fields.h5')
```

Comment:
Revert all diff in this file except for this section.
There should probably an assertion as well.

Reply:
Done.